core/src/main/scripts/importer/validateData.py

```diff
@@ -2941,7 +2941,6 @@ def checkLine(self, data):
                     self.logger.error(
                             'Value in DFS_STATUS column is not 0:DiseaseFree, '
                             '1:Recurred/Progressed, 1:Recurred, 1:Progressed',
```

Comment:
I think we should still keep these values recommendations, I was wondering if you delete the comma at the end of this line, will that fix the problem too?
```suggestion
                            '1:Recurred/Progressed, 1:Recurred, 1:Progressed'
```